src/SprykerSdk/Zed/AppSdk/Business/Configuration/Builder/ConfigurationBuilder.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Zed\AppSdk\Business\Configuration\Builder;
+
+use Generated\Shared\Transfer\AppConfigurationRequestTransfer;
+use Generated\Shared\Transfer\AppConfigurationResponseTransfer;
+
+class ConfigurationBuilder implements ConfigurationBuilderInterface
+{
+    /**
+     * @var array<string>
+     */
+    protected $transferToAsyncApiTypeMap = [
+        'int' => 'integer',
+    ];
+
+    /**
+     * @param \Generated\Shared\Transfer\AppConfigurationRequestTransfer $appConfigurationRequestTransfer
+     *
+     * @return \Generated\Shared\Transfer\AppConfigurationResponseTransfer
+     */
+    public function appConfigurationCreate(AppConfigurationRequestTransfer $appConfigurationRequestTransfer): AppConfigurationResponseTransfer
+    {
+        $appConfigurationResponseTransfer = new AppConfigurationResponseTransfer();
+
+        $this->writeToFile(
```

Comment:
Where is the "Business" logic? You can not just write properties to the file. I expect that the `AppConfigurationRequestTransfer` contains all information you need to create a proper configuration file in *this* class.

Reply:
I moved business login from communication to business layer.